sys/kern/build.rs

```diff
@@ -292,7 +303,7 @@ fn translate_address(
     task_index: usize,
     address: OwnedAddress,
 ) -> u32 {
-    let key = RegionKey::Owned(task_index, address.region_name);
+    let key = RegionKey::Owned(task_index, 0, address.region_name);
```

Comment:
Is this right? It seems simpler than I'd expect.

Reply:
Yes, although it's a little subtle:

- Regions are populated in order from the `MultiRegionConfig`, so region 0 is going to be the first region in memory (at an address given by `MultiRegionConfig::base`)
- Regions are contiguous, so the offset can be taken from the first region's base address

I added a comment to that effect in 122b0a127318bfa586542f1dfce0bd927eeff04f